File: backend/src/main/java/com/galinho/backend/dto/Estoque/LoteProdutoCreate.java
package com.galinho.backend.dto.Estoque;

import java.time.LocalDateTime;

import jakarta.validation.constraints.Null;

public record LoteProdutoCreate (
    @Null Long id,
    int quantidade,
    LocalDateTime validade,
    String codigoLote,
    Long fornecedorId,
    Long produtoId 
) {
    
}

File: frontend/src/interfaces/LoteProdutoCreate.ts
import type { Fornecedor } from "./Fornecedor";
import type { Produto } from "./Produto";

export interface LoteProdutoCreate {
  id: null;
  quantidade: number;
  validade: string;
  codigoLote: string;
  fornecedorId: number;
  produtoId: number;
}
